internal/transactions/sign.go

```diff
@@ -54,22 +56,47 @@ func sign(
 	services *services.Services,
 	state *flowkit.State,
 ) (command.Result, error) {
-	filename := args[0]
-	payload, err := readerWriter.ReadFile(filename)
+	var payload []byte
+	var err error
+	yesOption := globalFlags.Yes
+	var filenameOrUrl string
+
+	if signFlags.FromRemoteUrl != "" {
+		filenameOrUrl = signFlags.FromRemoteUrl
+		payload, err = services.Transactions.GetRlp(filenameOrUrl)
+		yesOption = false // force user to view remote tx
```

Comment:
IMO it's better to error if `globalFlags.Yes` is true instead of silently ignoring it.